Jellyfin.Plugin.YoutubeMetadata/Utils.cs

```diff
@@ -78,7 +78,7 @@ public static async Task<string> SearchChannel (string query, IServerApplication
             ytd.Options.VerbositySimulationOptions.Simulate = true;
             ytd.Options.GeneralOptions.FlatPlaylist = true;
             ytd.Options.VideoSelectionOptions.PlaylistItems = "1";
-            ytd.Options.VerbositySimulationOptions.PrintField = "url";
```

Comment:
This should be left alone I believe. Why did you comment it out?

Reply:
It was removed from newer versions of the ytd library, and as such wouldn't compile for me with it present.

Reply:
```
> yt-dlp --version
2022.06.22.1
> yt-dlp -s --flat-playlist -I 1 -O url https://www.youtube.com/results?search_query=Josh+Strife+Hayes
https://www.youtube.com/channel/UCRWyPm7MrfotIYF8A8MGV3g
```

Not sure what you mean, it is still present in yt-dlp. Are you still using yt-dl? What won't compile exactly? Without url this function will not work.